js/src/admin/components/BasicsPage.js

```diff
@@ -42,8 +38,29 @@ export default class BasicsPage extends Page {
       this.displayNameOptions[identifier] = identifier;
     }, this);
 
+    this.slugDriverOptions = {};
+    Object.keys(app.data.slugDrivers).forEach((model) => {
+      this.fields.push(`slug_driver_${model}`);
+      this.slugDriverOptions[model] = {};
+
+      app.data.slugDrivers[model].forEach((option) => {
+        this.slugDriverOptions[model][option] = option;
```

Comment:
Are the keys getting localized somewhere ? I mean `default` and `idOnly` for example (same with display name drivers) They don't seem to be, at least the option's label not the key itself.

Reply:
Nope, and mail drivers aren't either. That's a limitation worth revisiting later IMO

Reply:
I see now, something like `app.translator.trans('core.admin.basics.slug_drivers.${option}')` wouldn't work, since extensions are adding drivers, and that means they couldn't add locales for the drivers anyway.

The solution would need to be more sophisticated. Yeah worth revisiting it later.